### PR TITLE
{bio}[GCCcore/11.3.0] CSBLAST v2.2.4

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Review by Micket — DISMISSED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/c/CSBLAST/CSBLAST-2.2.4-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+dependencies = [
+    ('sparsehash', '2.0.4'),
+]
+
+start_dir = 'src'
+
+build_cmd_targets = 'csblast csbuild'
+
+buildopts = '"FLAGS=-fpermissive  -std=c++11 "'
```
Comment (Micket) [inline easybuild/easyconfigs/c/CSBLAST/CSBLAST-2.2.4-GCCcore-11.3.0.eb:28]: is that double quotation here really correct? I would expect
```suggestion
buildopts = 'FLAGS=-fpermissive  -std=c++11 '
```

Comment (sassy-crick) [reply easybuild/easyconfigs/c/CSBLAST/CSBLAST-2.2.4-GCCcore-11.3.0.eb:28]: I think I set it at the wrong place. Without any double quotes I get:
```
ERROR EasyBuild crashed with an error (at easybuild/tools/build_log.py:111 in caller_info): cmd " make csblast csbuild -j 8 FLAGS=-fpermissive -s
td=c++11 " exited with exit code 2 and output:
make: invalid option -- '='
make: invalid option -- 'c'
make: invalid option -- '+'
make: invalid option -- '+'
make: invalid option -- '1'
make: invalid option -- '1'
Usage: make [options] [target] ...
```
However, it works like that:
```
buildopts = 'FLAGS="-fpermissive -std=c++11" '
```

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/5b46ecce0c5a6594e614a2829ad2929e for a full test report.

**Pull request merged**